skyVegasTests/gamePlayGameClient.spec.ts

```diff
@@ -1,35 +1,261 @@
-import { test, type Page } from '@playwright/test';
+import { test, type Page, expect } from '@playwright/test';
 import {
     launchGame,
     login,
-    validateConsoleMessages,
     startEventListener,
-    makeSpin,
+    firstSpin,
     readGames,
+    spin,
+    getValueFromConsoleMessages,
+    currencyStringToNumber,
 } from './utilities';
 import { ExpectedMessage } from '../types/expectedMessage';
 
 let page: Page;
 readGames().forEach((game) => {
-    const consoleMessages: string[] = [];
-
     test.describe(`Testing with text: ${game}`, () => {
-        test.beforeAll(async ({ browser }) => {
+        test.beforeEach(async ({ browser }) => {
+            test.setTimeout(300000);
             page = await browser.newPage();
             await login(page);
         });
 
-        test.afterAll(async () => {
+        test.afterEach(async () => {
             await page.close();
         });
 
-        test('gamePlay', async () => {
+        test('gamePlayForLoss', async () => {
+            const consoleMessages: string[] = [];
+            let numberOfSpins = 0;
+            startEventListener(page, consoleMessages);
+
+            await launchGame(page, game, consoleMessages);
+
+            const startBalanceConsole = await getValueFromConsoleMessages(
+                consoleMessages,
+                ExpectedMessage.BALANCE_UPDATE
+            );
+
+            const startBallanceScreen = currencyStringToNumber(
```

Comment:
small typo here

Reply:
Will make this change in the PR for GTX-441. We had to pull that work into this one as there were overlaps with helper functions that were needed.